cmd/zetaclientd/start.go

```diff
@@ -157,6 +160,25 @@ func start(_ *cobra.Command, _ []string) error {
 
 	go zetacoreClient.UpdateAppContextWorker(ctx, appContext)
 
+	keygen := appContext.GetKeygen()
```

Comment:
We should add `c.GetAllNodeAccounts()` to `UpdateAppContext()` and use that instead. That will also work during keygen.

Reply:
I think we should still use the Keygen object, as the node account list can be modified when we add an observer.

The newly added observer should not be allowed into the whitelist unless they are part of a tss ceremony (i.e. the keygen object )

Reply:
I agree , and I think that list should be the keygen List ,and not the nodeAccount list .
Becuase of the fact that the nodeAccount list might contain accounts which are not part of TSS yet. 

We might need to track this in a separate issue if we need to modify the keysign list as well .